examples/labs/orientation/orientation.pro

```diff
@@ -1,19 +1,35 @@
-
 TEMPLATE = app
-DEPENDPATH += .
-INCLUDEPATH += .
-
-CONFIG += qt3d
+TARGET = orientation
 
-SOURCES += main.cpp
+CONFIG += c++17
+QT += core gui widgets 3dcore 3drender 3dinput 3dextras 3dlogic
 
-SOURCES += orientationview.cpp
-HEADERS += orientationview.h
+# Sources / headers
+SOURCES += \
+    main.cpp \
+    orientationview.cpp
 
-HEADERS += orientation.h
+HEADERS += \
+    orientationview.h \
+    orientation.h
 
+# PS Move include/lib layout as before
+DEPENDPATH += .
+INCLUDEPATH += .
 DEPENDPATH += ../../../include
 INCLUDEPATH += ../../../include
 
-LIBS += -L../../../build/ -lpsmoveapi -lpsmoveapi_tracker -lopencv_highgui
+# NEW: where your psmove landed
+INCLUDEPATH += /usr/local/include
+# Sometimes headers live in a nested folder, add this too if needed:
+INCLUDEPATH += /usr/local/include/psmoveapi
```

Comment:
Not sure if this is a good idea, we have the headers above and link against a local build. The headers in `/usr/local` might very well be a different version. Remove this...